src/pkg/packager/actions/actions.go

```diff
@@ -114,6 +128,38 @@ retryCmd:
 				}
 			}
 
+			// If an output value is defined, parse the result and set it to values map.
+			for _, v := range action.SetValues {
```

Comment:
It's pretty hard to unit test actions since they shell out. It might be good to e2e test this part

Reply:
This is covered in e2e now